spec/schemas/_common.yaml

```diff
@@ -1205,32 +1205,32 @@ components:
       description: The amount of lag during upload between the remote segment store and the local store.
       properties:
         max:
-          description: The maximum amount of lag, in bytes, during the upload refresh between the remote segment store and the local store.
+          description: The maximum amount of lag, in bytes, during the upload refresh between the remote segment store and the local store. 
           $ref: '#/components/schemas/HumanReadableByteCount'
         max_bytes:
-          description: The maximum amount of lag, in bytes, during the upload refresh between the remote segment store and the local store.
+          description: The maximum amount of lag, in bytes, during the upload refresh between the remote segment store and the local store. 
           $ref: '#/components/schemas/ByteCount'
         total:
-          description: The total number of bytes that lagged during the upload refresh between the remote segment store and the local store.
+          description: The total number of bytes that lagged during the upload refresh between the remote segment store and the local store. 
           $ref: '#/components/schemas/HumanReadableByteCount'
         total_bytes:
-          description: The total number of bytes that lagged during the upload refresh between the remote segment store and the local store.
+          description: The total number of bytes that lagged during the upload refresh between the remote segment store and the local store. 
           $ref: '#/components/schemas/ByteCount'
       required:
         - max_bytes
         - total_bytes
     RemoteStoreDownloadStats:
       type: object
-      description: Statistics related to downloads to the remote segment store.
+      description: Any statistics related to downloads to the remote segment store. 
       properties:
         total_download_size:
-          description: The total amount of data download from the remote segment store.
+          description: The total amount of data download from the remote segment store. 
           $ref: '#/components/schemas/RemoteStoreUploadDownloadStats'
         total_time_spent:
-          description: The total duration, spent on downloads from the remote segment store.
+          description: The total duration, spent on downloads from the remote segment store. 
           $ref: '#/components/schemas/Duration'
         total_time_spent_in_millis:
-          description: The total duration, in milliseconds, spent on downloads from the remote segment store.
+          description: The total duration, in milliseconds, spent on downloads from the remote segment store. 
```

Comment:
"The total amount of time, in milliseconds, spent downloading [noun] from the remote segment store"?